AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.ts

```diff
@@ -44,6 +44,8 @@ export class IdentityDetailsComponent {
     public updatedTier?: TierOverview;
     public tier?: TierOverview;
 
+    private dialogRef: MatDialogRef<AssignQuotasDialogComponent> | undefined;
```

Comment:
Why do you define all other properties with `name?: Type`, and here you suddenly start with `name: Type | undefined`?

Reply:
I was preoccupied to make dialogRef work in the code and simply forgot to change it to name?: .